index.js

```diff
@@ -135,16 +135,14 @@ const sortScripts = onObject((scripts) => {
   const names = Object.keys(scripts)
   const prefixable = new Set()
 
-  const keys = names
-    .map((name) => {
-      const omitted = name.replace(/^(?:pre|post)/, '')
-      if (defaultNpmScripts.has(omitted) || names.includes(omitted)) {
-        prefixable.add(omitted)
-        return omitted
-      }
-      return name
-    })
-    .sort()
```

Comment:
Seems like this is the only LOC that should be in this diff now, no?

Reply:
I believe this is prettier moving the `.map` up a line since it no longer has a `.sort` to align with

Reply:
@keithamus does this sufficiently explain the diff?

Reply:
Why has the `index.d.ts` changed so much? Is it just that the copy in the main branch is old?